base_ir_values_security_adj/security/base_security.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+
+<odoo>
+    <data noupdate="1">
+        <delete model="ir.rule" id="base.ir_values_default_rule"/>
+
+        <record model="ir.rule" id="ir_values_default_rule">
```

Comment:
Let's create back the same rule, i.e. id="base.ir_values_default_rule"

Reply:
I think if I trying to create the same rule name: then first it will find external id which was deleted from above <delete function.

Also in base code for id having `base.ir_values_default_rule` noupdate=1 so that's why we need to remove update the rule.